extensions/cornerstone/src/init.js

```diff
@@ -4,30 +4,32 @@ import csTools from 'cornerstone-tools';
 import initCornerstoneTools from './initCornerstoneTools.js';
 import queryString from 'query-string';
 
-function fallbackMetaDataProvider (type, imageId) {
+function fallbackMetaDataProvider(type, imageId) {
   if (!imageId.includes('wado?requestType=WADO')) {
-    return
+    return;
   }
 
-  // If you call for an WADO-URI imageId and get no 
+  // If you call for an WADO-URI imageId and get no
   // metadata, try reformatting to WADO-RS imageId
   const qs = queryString.parse(imageId);
-  const wadoRoot = window.store.getState().servers.servers[0].wadoRoot
-  const wadoRsImageId = `wadors:${wadoRoot}/studies/${qs.studyUID}/series/${qs.seriesUID}/instances/${qs.objectUID}/frames/${qs.frame || 1}`;
+  const wadoRoot = window.store.getState().servers.servers[0].wadoRoot;
+  const wadoRsImageId = `wadors:${wadoRoot}/studies/${qs.studyUID}/series/${
+    qs.seriesUID
+  }/instances/${qs.objectUID}/frames/${qs.frame || 1}`;
 
   return cornerstone.metaData.get(type, wadoRsImageId);
 }
 
 // Add this fallback provider with a low priority so it is handled last
 cornerstone.metaData.addProvider(fallbackMetaDataProvider, -1);
 
-
 /**
  *
  * @param {object} configuration
  * @param {Object|Array} configuration.csToolsConfig
  */
-export default function init(configuration = {}) {
+export default function init(services, configuration = {}) {
```

Comment:
I'm wondering if we should change-up this signature.

Should we inject the ServicesManager, or all services?
Should we pass in an object instead of named parameters to reduce the chances of introducing breaking changes to extensions?

Reply:
Once we land on an ideal solution here, we can make sure the other Extension Module's have access to the services with a similar pattern

Reply:
I think we should change it to be an object instead of named parameters... About the manager instead of all services, do you think that the extension could leverage the manager itself somehow?

Reply:
I'm not sure.

If I receive `{ ServiceName, AnotherService, AndAnotherService, Configuration }`
Versus: `{ ServiceManager, Configuration }`

We can probably add some convenience methods to `ServiceManager` that verify if a service exists, or if it's implementation has been set, etc.